cumulus_library/cli.py

```diff
@@ -344,11 +347,28 @@ def run_cli(args: dict):
                             )
 
             elif args["action"] == "export":
+                if args["archive"]:
```

Comment:
I initially had a thought that maybe this should be a toplevel action instead of a flag on `export` but after thinking about it more, decided you had the right of it. Just talking out loud - in case you had thoughts on that too. But I think this makes sense.